docs/issue_tracking.md

```diff
@@ -0,0 +1,80 @@
+<!-- {% raw %} -->
+
+# Issue Tracking
+
+Issues are used to track all aspects of the project, from the roadmap, through
+to milestones and workstreams, to individual tasks / bugs.
+
+## HLSL Support project
+
+The [HLSL Support](https://github.com/orgs/llvm/projects/4) github project
+brings together issues in the [wg-hlsl](https://github.com/llvm/wg-hlsl) and
+[llvm-project](https://github.com/llvm/llvm-project) repos. This project is used
+to manage scheduling work, tracking size and priority estimates as well as
+workstreams and milestones.
+
+## Project Milestones
+
+In order to understand progress being made across the entire project, a set of
+Milestones are defined. We track these with issues rather than using github's
+first-class milestone feature since github milestones are set per-repo and we
+want to be able to link issues to milestones across repos.
+
+A milestone issue can be identified by the HLSL Support project's "Category"
+field being set to "Project Milestone".
+
+The current target date for hitting a milestone is in the "Target" field.
+
+The current list of milestones can be seen on the [Milestones
+view](https://github.com/orgs/llvm/projects/4/views/15). The [Roadmap
+view](https://github.com/orgs/llvm/projects/4/views/12) also provides a
+graphical view of the milestones over time.
+
+The main body for a Milestone issue is follows this format:
```

Comment:
This 'is' in the sentance reads weird...  Could be just me and not having my morning caffine yet.